manifests/repo.pp

```diff
@@ -4,8 +4,8 @@
 #
 #   include homebrew::repo
 class homebrew::repo (
-  $installdir   = $homebrew::config::installdir,
-  $min_revision = $homebrew::config::min_revision,
+  $installdir   = $homebrew::installdir,
+  $min_revision = $homebrew::min_revision,
```

Comment:
you can set in https://github.com/boxen/our-boxen/blob/master/hiera/common.yaml.example
```
homebrew::min_revision: 'fbc5fc3e92587b47ddd8d8a97eb8e3a138d72957'
homebrew::repo: 'homebrew/brew'
```

 and puppet will pickup your custom settings and apply them.